src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -89,12 +95,14 @@ private SpanBuilder getSpanBuilder(RoutingContext routingContext, String operati
         return spanBuilder;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
A comment what is deprecated might be useful. Also, if something should be removed later, an issue would be nice for tracking and link it in the comments.

Reply:
I've added a comment explaining the suppressions and raised https://github.com/strimzi/strimzi-kafka-bridge/issues/875 to remove the deprecated attributes.